spec/requests/create_work_spec.rb

```diff
@@ -347,7 +347,8 @@
             contact_email: '',
             abstract: '',
             license: License.license_list.first,
-            work_type: 'text'
+            work_type: 'text',
+            release: 'immediate'
```

Comment:
Needed the changes in this file to make the tests pass and after spending more than half my day fighting reform after having already had a working PR... I didn't care enough to put more time into this PR.